stripe/__init__.py

```diff
@@ -36,6 +43,44 @@
     os.path.dirname(__file__), "data", "ca-certificates.crt"
 )
 
+# Lazily initialized stripe.default_http_client
+default_http_client = None
+_default_proxy = None
+
+
+def ensure_default_http_client():
```

Comment:
The API requestor is the only thing that calls this method. Worth inlining the default client logic into it?

Reply:
Done in https://github.com/stripe/stripe-python/pull/1200/commits/458614d69533618d18ed0dc6629caaca31ab4b8a